tests/kernel/pipe/deprecated/pipe_api/src/test_pipe_fail.c

```diff
@@ -3,8 +3,6 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-
```

Comment:
@Mattemagikern - can you please drop unnecessary whitespace changes?

Reply:
Certainly! 

I think I got them all! :)